src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java

```diff
@@ -49,7 +49,7 @@ public void execute_invalidIndexUnfilteredList_throwsCommandException() {
 
     @Test
     public void execute_validIndexFilteredList_success() {
-        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+        showGuestAtIndex(model, INDEX_FIRST_PERSON);
```

Comment:
Should this be INDEX_FIRST_GUEST as well? Found in seedu.address.testutil.TypicalIndexes

Reply:
This should be refactored under Zong Xian's (your) part as it is this under the testutil package.